src/FieldLabel/Outlined/Label.js

```diff
@@ -64,6 +64,10 @@ const styles = css`
         position: relative;
     }
 
+    .required span::after {
+        content: '*';
```

Comment:
The other components that add the asterisk also have a space between the label and the asterisk, please add that.